docs/0___preamble.adoc

```diff
@@ -93,68 +93,65 @@ This document defines the Functional Mock-up Interface (FMI),
-are used that depend on the target environment
-(with exception of the data types of the target platform).
+In particular, it is possible to utilize several instances of a model and/or a co-simulation tool and to connect them together.
+The interfaces are independent of the target environment because no header files are used that depend on the target environment (with exception of the data types of the target platform).
 This allows generating one dynamic link library that can be utilized in any environment on the same platform.
-A model, a cosimulation slave or the coupling part of a tool,
-is distributed in one ZIP file called FMU (Functional Mockup Unit) that contains several files:
+A model, a cosimulation slave or the coupling part of a tool, is distributed in one ZIP file called FMU (Functional Mockup Unit) that contains several files:
 
 . An XML file contains the definitions of all exposed variables in the FMU and other static information.
-It is then possible to run the FMU on a target system without this information,
-in other words with no unnecessary overhead.
+It is then possible to run the FMU on a target system without this information, in other words with no unnecessary overhead.
 
 . All required model equations or the access to co-simulation tools are provided with a small set of easy-to-use C functions.
-A new caching technique allows a more efficient evaluation of
-the model equations than in other approaches.
+A new caching technique allows a more efficient evaluation of the model equations than in other approaches.
 These C functions can either be provided in source and/or binary form.
 Binary forms for different platforms can be included in the same FMU ZIP file.
 
 . The model equations or the co-simuation tool can be either provided directly in the FMU,
-or the FMU contains only a generic communication module that communicates
-with an external tool that evaluates or simulates the model.
-In the XML file, information about the capabilities of the FMU are present,
-for example to characterize the ability of a Co-Simulation slave to support advanced master algorithms such as the usage of variable communication step sizes,
-higher order signal extrapolation, or others.
-
-. Further data can be included in the FMU ZIP file,
-especially a model icon (bitmap file),
-documentation files,
-maps and tables needed by the FMU,
-and/or all object libraries or dynamic link libraries that are
-utilized.
-
-A growing set of tools supports FMI.
-The actual list of tools is available at: https://www.fmi-standard.org/tools
+or the FMU contains only a generic communication module that communicates with an external tool that evaluates or simulates the model.
+In the XML file, information about the capabilities of the FMU are present, for example to characterize the ability of a Co-Simulation slave to support advanced master algorithms such as the usage of variable communication step sizes, higher order signal extrapolation, or others.
```

Comment:
...information about the exposed model variables (e.g. parameters, inputs and outputs) and about the capabilities...